### PR TITLE
Icon theming fix

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What

Fixed theming of LocalPreview placeholder in calling composite and MessageStatusIndicator in components.

# Why

https://skype.visualstudio.com/SPOOL/_workitems/edit/2543821

# How Tested

On calling samples. 
image
image

Also created small MessageThread to show all message status indicator icons in both dark and light theme.
image
image

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook/stories/ControlBar/Buttons/ControlBarButton/__snapshots__/ControlBarButton.stories.storyshot [hunk 1]
```diff
@@ -38,6 +38,7 @@ exports[`storybook snapshot tests Storyshots UI Components/ControlBar/Buttons/De
           <span
             aria-hidden={true}
             className="root-span"
+            primaryFill="currentColor"
```
Comment (mgamis-msft) [inline packages/storybook/stories/ControlBar/Buttons/ControlBarButton/__snapshots__/ControlBarButton.stories.storyshot:41]: Not sure why these are being update when I run `rushx test -u` in packages/storybook. I went back to main and ran `rushx test -u` and this also happens.

Comment (JamesBurnside) [reply packages/storybook/stories/ControlBar/Buttons/ControlBarButton/__snapshots__/ControlBarButton.stories.storyshot:41]: I just pushed a fix that removed these, you'll run to run `rush update` to ensure storybook tests are being run with the latest `@fluent/react-icons` package to ensure these don't show up

Comment (mgamis-msft) [reply packages/storybook/stories/ControlBar/Buttons/ControlBarButton/__snapshots__/ControlBarButton.stories.storyshot:41]: Ah ok. Will do that. Thanks!

File: packages/react-composites/src/composites/CallComposite/LocalPreview.tsx [hunk 2]
```diff
@@ -38,6 +32,20 @@ export const LocalPreview = (): JSX.Element => {
     isLocalMicrophoneEnabled ? adapter.mute() : adapter.unmute();
   }, [adapter, isLocalMicrophoneEnabled]);
 
+  const theme = useTheme();
+  const onRenderPlaceholder = useCallback((): JSX.Element => {
+    return (
+      <Stack style={{ width: '100%', height: '100%', color: theme.palette.black }} verticalAlign="center">
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/CallComposite/LocalPreview.tsx:38]: nit: this should use `styles` or `className` for setting styling on a fluent element (using styles or className creates a css class for the element, setting `style` directly sets the style on the dom element inline)

**Review by JamesBurnside — APPROVED** [review]

File: packages/react-composites/src/composites/CallComposite/LocalPreview.tsx (hunk 2 above)
Comment (JamesBurnside) [reply packages/react-composites/src/composites/CallComposite/LocalPreview.tsx:38]: was it intentional this changed from `theme.palette.neutralPrimaryAlt` to `theme.palette.black` ?

Comment (JamesBurnside) [reply packages/react-composites/src/composites/CallComposite/LocalPreview.tsx:38]: From the figma looks like our other icons are `palette.neutralPrimary` (not black or alt): https://www.figma.com/file/72qJ03EOQExf12g5JTdWjQ/ACS-UI---POR?node-id=2136%3A8597

Comment (mgamis-msft) [reply packages/react-composites/src/composites/CallComposite/LocalPreview.tsx:38]: Good catch. I will change it to `palette.neutralPrimary`.

**Review by alcail — APPROVED** [review]

**Pull request merged**